src/components/SEBankIDQRCode.tsx

```diff
@@ -135,10 +139,9 @@ export default function SEBankIDQrCode(props: Props) {
   /**
    * Render QR codes
    */
-  useDraw(qrCode, {
-    canvas: canvasRef.current,
-    width: wrapperRef.current?.offsetWidth,
-    qrMargin: props.qrMargin
+  const qrDataURL = useDraw(qrCode, {
+    width: fullscreen ? (window.innerHeight ?? wrapperRef.current?.offsetWidth) : wrapperRef.current?.offsetWidth,
+    qrMargin: props.qrMargin ?? (fullscreen ? 2 : undefined)
```

Comment:
Would be nice to have some explanation of the magic number `2`.

Reply:
@fkj It's half of the default 4 (described elsewhere in the file)

Reply:
Will add a comment

Reply:
97a228c